### PR TITLE
playground: Rewrite the playground speed page

### PR DESCRIPTION
**Pull request by stefwalter**
This lets us use the speed page for testing.

 * Adds ability to read files for a speed test
 * Removes use of jQuery

### CODE REVIEW

**Comment by stefwalter** [comment]
By clicking buttons in the Read section you should be able to see the RSS memory for Cockpit spike, and the need for flow control.

File: pkg/playground/speed.js [hunk 1]
```diff
@@ -1,136 +1,179 @@
...
-        $("#normal").on("click", normal);
-        $("#sideband").on("click", sideband);
-        $("body").show();
+        document.getElementById("echo-normal").addEventListener("click", echo);
+        document.getElementById("echo-sideband").addEventListener("click", echo);
+        document.getElementById("read-normal").addEventListener("click", read);
+        document.getElementById("read-sideband").addEventListener("click", read);
+        document.getElementById("stop").addEventListener("click", stop);
+        window.setInterval(update, 500);
+        document.body.style.display = "block";
```
Comment (martinpitt) [inline pkg/playground/speed.js]: That looks a bit odd - aren't we usually using `hidden` and `element.removeAttribute("hidden")`?
But this is total bikeshedding of course, not not at all important.

Comment (stefwalter) [reply pkg/playground/speed.js]: Makes sense. I'll do that in the future. Let me know if i should do that here.

**Comment by stefwalter** [comment]
#9553 Uses this to implement its test that flow control works. I wanted to get this in early, so we can manually exhibit the problem via this page before #9553 lands.

**Comment by martinpitt** [comment]
If I enter a path to a big file (VM image) into the "read test" path, and use "normal channel", it works, and I get some ~ 2 MB/s read speed. But when I click on "sideband channel" I just get a popup "channel closed". That might be intentional, or get fixed in a followup PR, but I just wanted to point it out. (Could also be that I'm running cockpit-{ws,bridge} 171, not master?)

File: pkg/playground/speed.js (hunk 1 above)
Comment (martinpitt) [reply pkg/playground/speed.js]: No need to reset the tests and spend time on this IMHO; I just caught my eye, as it's a rather unusual way of showing an element.

**Pull request merged**